src/schema/v2/Feature/FeatureImageType.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLNonNull,
+} from "graphql"
+import { Gravity } from "@artsy/quartz"
+import { scale } from "proportional-scale"
+import { ResolverContext } from "types/graphql"
+import proxy from "schema/v2/image/proxies"
+import { ScaledImage, TScaledImage } from "../image"
+
+export const FeatureImageType = new GraphQLObjectType<
+  Gravity.Feature,
```

Comment:
Does anyone have strong opinions on type naming conventions? I should probably rename this to be `Gravity.TFeature` for clarity?

Reply:
Why `TFeature`?